cfme/services/myservice/ui.py

```diff
@@ -174,7 +174,6 @@ def is_displayed(self):
         return (
             self.in_myservices and
             self.myservice.is_opened and
-            self.myservice.tree.currently_selected == self.context['object'].name and
```

Comment:
@niyazRedhat What is the error that you are getting here? `currently_selected` should work for this Accordion, check the Accordion name on L54 of this file. I think that we should keep this check so that the page is associated with the service name.

Reply:
@niyazRedhat why are you removing this?

Reply:
@john-dupuy @digitronik  I made changes please have a look.